.gitlab-ci.yml

```
@@ -326,15 +326,15 @@ test:helm_chart_upgrade_from_previous_lts_to_current_lts:
   script:
     - tests/ci-make-deps.sh
     - |
-      echo "INFO - installing mender tag ${MENDER_PREVIOUS_LTS_RELEASE_TAG:-mender-3.3.2} with helm chart version ${MENDER_LTS_HELM_VERSION:-5.0.1}"
+      echo "INFO - installing mender tag ${MENDER_PREVIOUS_LTS_RELEASE_TAG:-mender-3.6.5} with helm chart version ${MENDER_LTS_HELM_VERSION:-5.0.1}"
```

Could you also switch to the latest helm chart? Currently it's `5.9.0`. This because the previous LTS was compatible up to the 5.0 only, but now the previous LTS (`3.6`) is compatible with the latest helm chart.
```suggestion
      echo "INFO - installing mender tag ${MENDER_PREVIOUS_LTS_RELEASE_TAG:-mender-3.6.5} with helm chart version ${MENDER_LTS_HELM_VERSION:-5.9.0}"
```